src/components/eci/Country.js

```diff
@@ -77,6 +77,9 @@ const Country = (props) => {
   }, [register]);
 
   if (props.list === false) return null;
+
+  const isWindows = navigator.userAgent.indexOf("Win") > -1 ? true : false;
```

Comment:
@1v4n4, can you move it to src/hooks/useDevice AND wrap it into a custom hook (same way it is done for useIsMobile with useEffect (rationale, we shouldn't assume navigator is set, eg for unit tests)